src/content/content.e2e-spec.ts

```diff
@@ -0,0 +1,34 @@
+import * as request from 'supertest';
+import { Test } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import { ContentModule } from './content.module';
+import { ContentTypeModule } from '../content-type/content-type.module';
+
+describe('Cats', () => {
+  let app: INestApplication;
+
+  beforeAll(async () => {
+    const moduleRef = await Test.createTestingModule({
+      imports: [ContentTypeModule, ContentModule],
+    }).compile();
+
+    app = moduleRef.createNestApplication();
```

Comment:
let's sync up on the call about it. we probably don't have to require the app in tests as the app is going to be running in the docker container

Reply:
wait until something on port 3000